stories/LineChart.stories.js

```diff
@@ -168,6 +168,26 @@ storiesOf('LineChart', module)
     }
     return <ResizingChart />;
   })
+  .add('Stretchy', () => (
+    <React.Fragment>
+      <p>The chart should resize with resizing the window</p>
+      <div
+        style={{
+          width: '100%',
+          height: 'calc(100vh - 100px)',
+          border: '2px solid red',
+        }}
+      >
+        <DataProvider
+          defaultLoader={staticLoader}
+          baseDomain={staticBaseDomain}
+          series={[{ id: 1, color: 'steelblue' }, { id: 2, color: 'maroon' }]}
+        >
+          <LineChart />
+        </DataProvider>
+      </div>
+    </React.Fragment>
+  ))
```

Comment:
Isn't this story a duplicate of `Full-size` story?

Reply:
Yeah, more or less. Removed.